scripts/linux.sh

```diff
@@ -12,7 +12,7 @@ get_uuids() {
 }
 
 get_mountpoints() {
-  grep "^$1" /proc/mounts | cut -d ' ' -f 2
+  grep "^$1" /proc/mounts | cut -d ' ' -f 2 | sed 's,\\040, ,g'
```

Comment:
It's not `0x40`, it's `040` in octal, i.e. `0x20` in hex or `32` in decimal, or `' '` in ASCII ;-)
This is because `/proc/mounts` is space-separated, and so it needs to use some escape-sequence when a mountpoint itself contains a space character.